Tests/TestPATs/FixturePAT.swift

```diff
@@ -1,162 +1,235 @@
 import MockoloFramework
 
-let patNameCollision = """
-/// @mockable
-protocol Foo {
-associatedtype T
-}
+@Fixture enum simplePat {
+    protocol Foo {}
```

Comment:
```suggestion
    /// @mockable
    protocol Foo {}
```

Reply:
This is ok because `FooMock` is provided in "parent mock".
(I'm not sure what the original test want to ensure)

Reply:
"parent mock" is a crazy complicated feature of mockolo. I cannot describe the complete feature about it and it is not documented.

Reply:
Thank you for your explanation. I misunderstood.